packages/frontapp/src/app/components/BookInfo/index.tsx

```diff
@@ -365,13 +557,22 @@ const BookInfo: FC<IBookInfoProps> = ({
                 />
               ) : null}
             </>
+          ) : editing ? (
+            <WrapperButtons>
+              <StyledButton value="Save changes" onClick={editInformation} />
+              <StyledButton
+                value="Cancel changes"
+                transparent
+                onClick={discardChanges}
+              />
+            </WrapperButtons>
           ) : (
             <WrapperButtons>
               <StyledButton
                 value="Edit information"
                 transparent
                 svgComponent={<Edit />}
-                onClick={editInformation}
+                onClick={() => setEditing(true)}
```

Comment:
Better create handle function because your file has a lot of rows and it can be difficult to find functions like this one if we would like to change something in the future